### PR TITLE
Add rep additional contact details email address section and route

### PR DESCRIPTION
**Pull request by neil-stephen-mcgonigle**

### CODE REVIEW

**Comment by neil-stephen-mcgonigle** [comment]
Apologies going to make a change.

File: src/lib/resource/sections/rep-email-address.js [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+        schema: {
+            $schema: 'http://json-schema.org/draft-07/schema#',
+            type: 'object',
+            additionalProperties: false,
+            properties: {
+                'q-rep-email-address': {
+                    type: 'string',
+                    title: 'Enter your email address',
+                    description:
+                        'We may use this to contact you if we need to clarify something in this application form (optional).',
```
Comment (Julieallan) [inline src/lib/resource/sections/rep-email-address.js:14]: @neil-stephen-mcgonigle I have spoken to Graham re the inclusion of the word `form` and he agrees that it's not needed. Can you remove it so sentence reads `We may use this to contact you if we need to clarify something in this application (optional).` I'll update the story with a comment and Graham will update the prototype. Thank you

File: src/lib/resource/sections/rep-email-address.js [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+                    maxLength: 50,
+                    format: 'email',
+                    errorMessage: {
+                        maxLength: 'Email address must be 50 characters or less',
+                        format: 'Enter your email address, for example john.smith@email.com'
```
Comment (Julieallan) [inline src/lib/resource/sections/rep-email-address.js:19]: Can we use 'Enter an email address in the correct format, like name@example.com'. Thanks

**Review by Julieallan — CHANGES_REQUESTED** [review]

File: src/lib/resource/sections/rep-email-address.js (hunk 1 above)
Comment (neil-stephen-mcgonigle) [reply src/lib/resource/sections/rep-email-address.js:14]: Sorry Julie, 'form' is still there in applicant-enter-your-email-address

File: src/lib/resource/sections/rep-email-address.js (hunk 2 above)
Comment (neil-stephen-mcgonigle) [reply src/lib/resource/sections/rep-email-address.js:19]: Done

**Review by Julieallan — APPROVED** [review]

**Pull request merged**